app/assets/main/react_components/footprint_form/Title.jsx

```diff
@@ -1,8 +1,8 @@
 import React from 'react';
 
-const Title = ({text}) => {
+const Title = ({text, custom_style}) => {
   return (
-    <h2 className="heading my-4">{text}</h2>
+    <h2 className={"heading my-4 " + ((!(custom_style === undefined)) ? custom_style : "")} >{text}</h2>
```

Comment:
Reagerade på skrivsättet, men kanske bara är jag? Vad känner ni @teolenn @e-ry ? Brukar skriva som förslaget. Obs ej testkört så kanske missförstår syntaxen.

```suggestion
    <h2 className={"heading my-4 " + (custom_style !== undefined ? custom_style : "")} >{text}</h2>
```

Reply:
Helt rimligt, kommer nog från en "ah juste det är tvärt om!" och inte för att det är bäst